lib/symbol-store.coffee

```diff
@@ -174,6 +174,7 @@ class SymbolStore
 
     for bufferRow in [bufferRange.start.row..bufferRange.end.row] by 1
       tokenizedLine = tokenizedLines[bufferRow]
```

Comment:
@nathansobo it's curious that the range reports a buffer row that does not have a tokenized line. Looks like it's happening in onWillChange: https://github.com/atom/autocomplete-plus/issues/477, and in the initial list build: https://github.com/atom/autocomplete-plus/issues/478 Not sure. But this check should fix the error.